libraries/ride-fuzzer-lib/src/main/java/com/adobe/ride/libraries/fuzzer/MetadataFuzzer.java

```diff
@@ -140,45 +148,142 @@ private void initializeFuzzer(String serviceName, ModelObject objectToBeFuzzed,
 
     // Populate global properties
     this.entity = objectToBeFuzzed;
+    this.entity.buildValidModelInstance();
+
     this.requestMethod = method;
     this.serviceName = serviceName;
-    this.properties = this.entity.getModelProperties();
-    this.instance = this.entity.getObjectMetadata();
+    this.properties = ModelObject.getObjectNodeProperties(entity, this.entity.getModel());
```

Comment:
Previously, this code only worked with root object-style schemas.  This now allows for root array-style schemas.